### DISCUSSION TITLE
第3节课笔记(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#180] [笔记]
12班第3节课的笔记提交在这里哦~

**Comment by Jeff-Huang-SHU** [comment]
https://www.yuque.com/u34036176/dxc2up/ml5uy3sg2not0iib?singleDoc# 《Langchain+InternLM》

**Comment by siuserjy** [comment]
https://blog.csdn.net/weixin_46334272/article/details/135459509

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135495759

**Comment by ltstar** [comment]
[课程3.md](https://github.com/InternLM/tutorial/files/13896978/3.md)

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec3/

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/1a1dc7130573423eaccd3c58c566e2b0

**Comment by Smiling-Weeping-zhr** [comment]
https://www.cnblogs.com/smiling-weeping-zhr/p/17963782

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135584563?spm=1001.2014.3001.5502

**Comment by al2h36o30s3** [comment]
https://zhuanlan.zhihu.com/p/677768856

**Comment by al2h36o30s3** [comment]
https://zhuanlan.zhihu.com/p/677768856

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/UpBvdGF1Eoz5UHx4yaicfnyangg

**Comment by JiangXiaobai00** [comment]
https://d0elb2210mg.feishu.cn/docx/PWPldeCCsoKcxmxWyelcLaP9n4f?from=from_copylink

**Comment by ShenFeilang** [comment]
https://shenfeilang.notion.site/InternLM-7ea839647f78412681035f29a0ae58e1?pvs=4
